apps/docs/src/app/core/component-docs/button/examples/button-toggled-example.component.html

```diff
@@ -0,0 +1,7 @@
+<button fd-button toggled label="Standard Button"></button>
+<button fd-button toggled label="Emphasized Button" fdType="emphasized"></button>
+<button fd-button toggled label="Ghost Button" fdType="ghost"></button>
+<button fd-button toggled label="Positive Button" fdType="positive" ariaLabel="Positive"></button>
+<button fd-button toggled label="Negative Button" fdType="negative" ariaLabel="Negative"></button>
+<button fd-button toggled label="Attention Button" fdType="attention" ariaLabel="Attention"></button>
+<button fd-button toggled label="Transparent Button" fdType="transparent" ariaLabel="Transparent"></button>
```

Comment:
Would it be reasonable to toggle "toggled" state on click here? Just so the example is more explicit?